test/hook.afterPackageCreateDeploymentArtifacts.test.js

```diff
@@ -74,4 +74,254 @@ describe('Serverless warmup plugin after:deploy:deploy hook', () => {
 
     expect(fs.remove).not.toHaveBeenCalled()
   })
+
+  it('Should package only the lambda handler by default', async () => {
+    const mockProvider = { request: jest.fn(() => Promise.resolve()) }
+    const serverless = getServerlessConfig({
+      getProvider () { return mockProvider },
+      service: {
+        custom: {
+          warmup: {
+            enabled: true
+          }
+        },
+        functions: { someFunc1: { name: 'someFunc1' }, someFunc2: { name: 'someFunc2' } }
+      }
+    })
+    const plugin = new WarmUp(serverless, {})
+
+    await plugin.hooks['after:package:initialize']()
+
+    expect(plugin.serverless.service.functions.warmUpPlugin)
+      .toEqual(getExpectedFunctionConfig({
+        package: {
+          individually: true,
+          exclude: ['**'],
+          include: ['_warmup/**']
+        }
+      }))
+  })
+})
+
+it('Should use the package exclusions from options if present', async () => {
+  const mockProvider = { request: jest.fn(() => Promise.resolve()) }
+  const serverless = getServerlessConfig({
+    getProvider () { return mockProvider },
+    service: {
+      custom: {
+        warmup: {
+          enabled: true,
+          package: {
+            individually: true,
+            exclude: ['../**']
+          }
+        }
+      },
+      functions: { someFunc1: { name: 'someFunc1' }, someFunc2: { name: 'someFunc2' } }
+    }
+  })
+  const plugin = new WarmUp(serverless, {})
+
+  await plugin.hooks['after:package:initialize']()
+
+  expect(plugin.serverless.service.functions.warmUpPlugin)
+    .toEqual(getExpectedFunctionConfig({
+      package: {
+        individually: true,
+        include: ['_warmup/**'],
+        exclude: ['../**']
+      }
+    }))
+})
+
+it('Should use the package inclusions from options if present', async () => {
+  const mockProvider = { request: jest.fn(() => Promise.resolve()) }
+  const serverless = getServerlessConfig({
+    getProvider () { return mockProvider },
+    service: {
+      custom: {
+        warmup: {
+          enabled: true,
+          package: {
+            individually: true,
+            exclude: ['../**'],
+            include: ['test/**']
+          }
+        }
+      },
+      functions: { someFunc1: { name: 'someFunc1' }, someFunc2: { name: 'someFunc2' } }
+    }
+  })
+  const plugin = new WarmUp(serverless, {})
+
+  await plugin.hooks['after:package:initialize']()
+
+  expect(plugin.serverless.service.functions.warmUpPlugin)
+    .toEqual(getExpectedFunctionConfig({
+      package: {
+        individually: true,
+        exclude: ['../**'],
+        include: ['test/**', '_warmup/**']
+      }
+    }))
+})
+
+it('Should use the package inclusions from options with the warmup folder', async () => {
```

Comment:
Rename to `Should not duplicate the warmup folder inclusion even if manually included`